### PR TITLE
dev-python/pymad: version bump to 0.10, add py3 support, update metadata

### PR DESCRIPTION
**Pull request by wimmuskee**
Not sure about the entry in package.mask, probably make it version specific on 0.6, or remove it entirely and wait on stable 0.10.

### CODE REVIEW

File: dev-python/pymad/pymad-0.10.ebuild [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
```
Comment (SoapGentoo) [inline dev-python/pymad/pymad-0.10.ebuild]: remove py2 while at it, so we don't have to revisit it

File: dev-python/pymad/pymad-0.10.ebuild [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+DEPEND="media-libs/libmad"
+RDEPEND="${DEPEND}"
+S="${WORKDIR}/${PN}-version-${PV}"
+
+python_configure_all() {
+	set -- "${PYTHON}" config_unix.py --prefix "${EPREFIX}/usr"
+	echo "${@}"
+	"${@}" || die "Configuration failed"
+}
```
Comment (SoapGentoo) [inline dev-python/pymad/pymad-0.10.ebuild]: is this needed at all?

File: dev-python/pymad/pymad-0.10.ebuild (hunk 1 above)
Comment (wimmuskee) [reply dev-python/pymad/pymad-0.10.ebuild]: done

File: dev-python/pymad/pymad-0.10.ebuild (hunk 2 above)
Comment (wimmuskee) [reply dev-python/pymad/pymad-0.10.ebuild]: no, compared installs with and without, no difference

**Pull request closed without merging**